mobile-first/index.html

```diff
@@ -0,0 +1,94 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  
+  <title>Great Idea - Responsive I</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body class="container">
+<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+  <nav class="navigation">
+    <a href="services.html">Services</a>
+    <a href="">Product</a>
+    <a href="">Vision</a>
+    <a href="">Features</a>
+    <a href="">About</a>
+    <a href="">Contact</a>
+
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+
+  <header>
+    <div class="main-heading">
+      <h1>Innovation
+        <br> On
+        <br> Demand
+      </h1>
+      <button class="get-started">
+            Get Started
+      </button>
+    </div>
+    <img class="header-image" src="img/header-img.png" alt="Image of a code snippet.">
+
+  </header>
+
+  <div class="first-section">
+    <section class="features-about">
+      <h3>Features</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis</p>
+    </section>
+    <section class="features-about">
+      <h3>About</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+  </div>
+
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+  <div class="second-section">
+    <section class="spv">
+      <h3>Services</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+
+    <section class="spv">
+      <h3>Product</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+
+    <section class="spv">
+      <h3>Vision</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+  </div>
+
+  <section class="contact">
+    <h3>Contact</h3>
+    <br> 123 Way 456 Street
+    <br> Somewhere, USA
+    <br>
+    <br> 1 (888) 888-8888
+    <br>
+    <br> sales@greatidea.io
+  </section>
```

Comment:
Try to make sure all your content is wrapped in a tag. Each of these lines could be a paragraph or header, for example.